src/sentry/tagstore/snuba/backend.py

```diff
@@ -386,6 +370,14 @@ def get_groups_user_counts(self, project_id, group_ids, environment_id):
         result = snuba.query(start, end, ['issue'], None, filters, aggregations)
         return defaultdict(int, {k: v for k, v in result.items() if v})
 
+    def get_group_event_ids(self, project_id, group_id, environment_id, tags):
+        # This method is not implemented since the `event.id` column doesn't
+        # exist in Snuba.
```

Comment:
I don't think this is the right level of abstraction to implement event search ([see the calling code here](https://github.com/getsentry/sentry/blob/d0b8c41611ecdf81742a1ba6806b881d73c1c5d2/src/sentry/api/endpoints/group_events.py#L31-L114)) and I thinking trying to implement this solely for API compatibility is going to cause more problems/confusion than it solves in the short term so I'd rather just mark this as not implemented in this backend and come back to it.